packages/components/src/mobile/color-settings/index.native.js

```diff
@@ -176,18 +195,45 @@ function ColorSettings( {
 						isGradientColor={ isGradientColor }
 						currentSegment={ currentSegment }
 						isCustomScreen={ isCustomScreen }
-						onCustomPress={ () => {
-							onCustomScreenToggle( true );
-						} }
+						onCustomPress={ onCustomPress }
 						shouldEnableBottomSheetScroll={
 							shouldEnableBottomSheetScroll
 						}
 						defaultSettings={ defaultSettings }
 					/>
+					{ isCustomGadientShown && (
+						<>
+							<View style={ horizontalSeparatorStyle } />
```

Comment:
Maybe e could use `separatorType` instead? :)

Reply:
Unfortunately we cannot, because even _fullWidth_ separator from cell is providing a separator with horizontal paddings, since it takes `100%` of padded `PanelBody`.